<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWidget</class>
 <widget class="QWidget" name="TestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="PushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Click me</string>
   </property>
  </widget>
  <widget class="ComboBox" name="languageComboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>94</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="ComboBox" name="themeComboBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>94</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="languageLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="themeLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>themeLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
